### PR TITLE
adding support for llama3 summarization prompt

### PR DESCRIPTION
**Pull request by madhurprash**
This PR contains the config file update and prompt template update for summarization use cases. This prompt template is specifically tested on large synthetic data that is meant to be summarized as a part of the benchmarking test.

### CODE REVIEW

**Comment by madhurprash** [comment]
added config files for g6e.48xl, g6e.24xl, inf2.48xl update for llama3.1 70b. All are tested.

File: src/fmbench/configs/llama3.1/70b/config-llama3-1-70b-g6e.48xl-deploy-ec2.yml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    payload_file: payload_en_500-1000.jsonl
+  - language: en
+    min_length_in_tokens: 1000
+    max_length_in_tokens: 2000
+    payload_file: payload_en_1000-2000.jsonl
+  - language: en
+    min_length_in_tokens: 2000
+    max_length_in_tokens: 3000
+    payload_file: payload_en_2000-3000.jsonl
+  - language: en
```
Comment (aarora79) [inline src/fmbench/configs/llama3.1/70b/config-llama3-1-70b-g6e.48xl-deploy-ec2.yml:118]: Can you remove this fragment from all files.

File: src/fmbench/configs/llama3.1/70b/config-llama3-1-70b-g6e.48xl-deploy-ec2.yml [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    min_length_in_tokens: 3000
+    max_length_in_tokens: 4000
+    payload_file: payload_en_3000-4000.jsonl
+  - language: en
+    min_length_in_tokens: 3000
+    max_length_in_tokens: 3840
+    payload_file: payload_en_3000-3840.jsonl
+  - language: en
```
Comment (aarora79) [inline src/fmbench/configs/llama3.1/70b/config-llama3-1-70b-g6e.48xl-deploy-ec2.yml:126]: remove please.

File: src/fmbench/configs/llama3.1/70b/config-llama3-1-70b-g6e.24xl-deploy-ec2.yml [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    # FMBench comes packaged witsh multiple inference scripts, such as scripts for SageMaker
+    # and Bedrock. You can also add your own. This is an example for a rest DJL predictor
+    # for a llama3-70b-instruct deployed on ec2
+    inference_script: ec2_predictor.py
+    # This section defines the settings for Amazon EC2 instances
+    ec2:
+      # The following line specifies the runtime and Inferentia settings for the instance
+      # '-u djl' tells the container to use the Deep Java Library (DJL) runtime
+      # '--device /dev/neuron0 ... /dev/neuron5' specifies the Inferentia devices to be used
+      gpu_or_neuron_setting: --gpus all --shm-size 12g
```
Comment (aarora79) [inline src/fmbench/configs/llama3.1/70b/config-llama3-1-70b-g6e.24xl-deploy-ec2.yml:190]: we dont need this parameter and the comments above it, please remove from all files.

**Review by aarora79 — CHANGES_REQUESTED** [review]

**Review by aarora79 — APPROVED** [review]

**Pull request merged**